modules/preferences/main/index.coffee

```diff
@@ -284,10 +284,8 @@ class Preferences extends hx.EventEmitter
   supportedTimezones: option 'supportedTimezones'
   timezoneOffsetLookup: option 'timezoneOffsetLookup'
 
-  applyTimezoneOffset: (date, offset) ->
-    offset ?= @_.timezoneOffsetLookup(@timezone(), date.getTime()) || 0
-    utc = date.getTime() + (date.getTimezoneOffset() * 60000)
-    new Date(utc + offset * 60 * 60 * 1000)
+  applyTimezoneOffset: (date) ->
+    new Date(date.getTime() + (date.getTimezoneOffset() * 60000))
```

Comment:
The behaviour of the api seems to have changed a bit here - it's no longer using the timezone from the preferences and no longer lets you optionally pass in a different offset.

I wonder if `executeFunctionInAllTimeZones` just needs updating to set up `preferences.timezone` and `preferences.timezoneOffsetLookup`?